gitlab/v4/cli.py

```diff
@@ -337,7 +337,9 @@ def _populate_sub_parser_by_class(
 
 def extend_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
     subparsers = parser.add_subparsers(
-        title="object", dest="gitlab_resource", help="Object to manipulate."
+        title="GitLab resource",
```

Comment:
Maybe we could do `resource`, as the lowercase titles from argparse are hardcoded and we already describe it in the help?
```suggestion
        title="resource",
```
Then it cascades kind of like this - `usage:`, `options:`, `optional arguments:`, `resources:`, `actions:`.

Reply:
Sure. Will do!

Reply:
Done.